tests/integration/test_charm.py

```diff
@@ -83,13 +84,19 @@ async def test_s3_storage(
     )
     channel = response.json()
 
-    # upload a file
+    # create a test file
+    tmp_path.mkdir(exist_ok=True)
```

Comment:
I don't think this is necessary, the path should already exist

Reply:
Removed